tests/test_urls_plugin.py

```diff
@@ -46,6 +46,50 @@ def test_find_urls():
             assert text.index(end) == f"1.{10 + len(url)}"
 
 
+# test if url gets splitted at quote or backtick
+def test_url_split_at_quote():
+    text = tkinter.Text(get_main_window())
+    urls = [
+        "https://example.com/single`quote",
+        "https://example.com/double`quote",
+        "https://example.com/backtick`something",
+    ]        
+    test_cases = """\
+          URL
+          URL bla bla
+"See also URL"
+         'URL bla'
```

Comment:
I don't think we need to copy/paste all this. Just add a line like this to the other test above:
```
         `URL`
```

You will also need to run `black tests/test_urls_plugin.py`